dev-docs/source/ToolsOverview.rst

```diff
@@ -285,3 +285,33 @@ For example, to convert all loops classified as *risky* or above, we would appen
 
     ;-config={CheckOptions: [ {key: modernize-loop-convert.MinConfidence, value: risky} ]}
 
+CMake-format
+---------------------
+
+`CMake-format <https://github.com/cheshirekow/cmake_format/>`__ is a tool which is used to format individual ``CMakeLists.txt`` to make them easier to read. 
+The package can be installed using ``pip install cmake_format`` or ``sudo pip install cmake_format``.
+
+To use cmake-format on a specific ``CMakeLists.txt`` file in the command line run 
+
+.. code::
+
+	python -m cmake_format -c /path/to/mantid/.cmake-format.json -i /path/to/CMakeLists.txt
+	
+This will format the file using the config file in the mantid directory.
+
+There is an official visual studio extension, details of which can be found `here <https://marketplace.visualstudio.com/items?itemName=cheshirekow.cmake-format>`__.
+
+To format on all the CMakeLists run the following from within the mantid directory:
+
+.. code::
+
+	import os
+
+	dir = os.getcwd()
+	for path, subdirs, files in os.walk(dir):
+		for file in files:
+			if file.endswith("CMakeLists.txt"):
+				cmakefile = path + "\\" + file
```

Comment:
I don't think this would work on all operating systems because of the windows backslash / ubuntu forward slash. Could we change this either to `cmakefile = os.path.join(path, file)` or mention that this is how to format CMakelists on windows?